src/GeneralStore/JS/createGeneralStoreName.ts

```diff
@@ -128,14 +133,20 @@ setup.createGeneralStoreName = (town, generalStore) => {
       break
     case 6:
       generalStore.name = lib.toTitleCase(`The ${noun} and ${family.toUpperFirst()}`)
+      // @ts-ignore
       generalStore.assistant = setup.createNPC(town, fam[family])
+      // @ts-ignore
       setup.createRelationship(town, associatedNPC, generalStore.assistant, family,
+        // @ts-ignore
         town.npcRelations[generalStore.assistant.key].filter(r => r.targetNpcKey === associatedNPC.key)[0]?.relation)
       break
     case 7:
       generalStore.name = lib.toTitleCase(`${associatedNPC.firstName} and ${family.toUpperFirst()}`)
+      // @ts-ignore
       generalStore.assistant = setup.createNPC(town, fam[family])
+      // @ts-ignore
       setup.createRelationship(town, associatedNPC, generalStore.assistant, family,
+        // @ts-ignore
         town.npcRelations[generalStore.assistant.key].filter(r => r.targetNpcKey === associatedNPC.key)[0]?.relation)
```

Comment:
Extract to a variable, since it's such a long line. Or even better, to a function since it's repeated in multiple places.